apps/docs/content/components/CTAForm.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: CTAForm
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/CTAForm/CTAForm.tsx
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=1377%3A30754'
+storybook: '/brand/storybook/?path=/story/components-CTAForm--playground'
+description: Use CTAForm to create a form that is used to collect minimal information from users.
+---
+
+import {
+  CTAForm,
+  InlineLink,
+  FormControl,
+  TextInput,
+  Checkbox,
+} from '@primer/react-brand'
+
+```js
+import {CTAForm} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<CTAForm>
+  <CTAForm.Input>
+    <FormControl required size="large" fullWidth>
+      <FormControl.Label>Your Work Email Address</FormControl.Label>
+      <TextInput placeholder="name" />
+    </FormControl>
+  </CTAForm.Input>
+  <CTAForm.Confirm>
+    <FormControl required size="large" fullWidth>
+      <FormControl.Label>
+        I agree to the <InlineLink href="#">Privacy Policy</InlineLink> and{' '}
+        <InlineLink href="#">Terms of Service</InlineLink>
+      </FormControl.Label>
+      <Checkbox name="confirm" />
+    </FormControl>
+  </CTAForm.Confirm>
+  <CTAForm.Action>Subscribe</CTAForm.Action>
+</CTAForm>
+```
+
```

Comment:
Nice to have: some additional examples for controlled / uncontrolled mode, so it's super clear and easy how to use this pattern.

Reply:
What were you imagining here for this.

Reply:
Similar to your test example, where you provide a submission handler that can trigger an alert or similar. E.g from PRC for controlled mode: https://primer.style/react/CheckboxGroup#using-onchange-handlers

Reply:
Thanks for the example I will add that in.